ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```diff
@@ -20,16 +20,12 @@
   tags:
     - skip_ansible_lint
 
-- name: Install xcode-tools
-  script: scripts/install-xcode.sh
-  when: xcode.rc > 1
```

Comment:
Homebrew automatically installs the command line tools if required. Reduces code to maintain.